test/spec/integration_spec.js

```diff
@@ -0,0 +1,30 @@
+describe('Integration Spec', function(){
+    var rules = customRules;
+
+    beforeEach(function() {
+        jasmine.getFixtures().fixturesPath = '/base/';
+    });
+
+    it('rules can be configured with axe-core', function(){
+        axe.configure(rules);
+        configuredRules = axe.getRules(['edx-custom']);
```

Comment:
Same `var` recommendation as above for `configuredRules ` and `configuredRuleIds`.